pgcli/main.py

```diff
@@ -260,7 +261,8 @@ def prompt_tokens(cli):
         application = Application(style=style_factory(self.syntax_style, self.cli_style),
                                   layout=layout, buffer=buf,
                                   key_bindings_registry=key_binding_manager.registry,
-                                  on_exit=AbortAction.RAISE_EXCEPTION)
+                                  on_exit=AbortAction.RAISE_EXCEPTION,
+                                  ignore_case=True)
```

Comment:
This causes search to be case insensitive. I guess that's what we want, because the autocompleter is also case insensitive.